.gitpod.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM gitpod/workspace-full
+
+RUN bash -c ". .nvm/nvm.sh \
+             && nvm install v12 && nvm alias default v12 \
+             && nvm use default && npm i -g yarn && alias near='yarn near'" \
```

Comment:
` alias near='yarn near'` why is this needed?

Reply:
So that users can use `near` without installing it globally.

Reply:
feel like it can be confusing. Why not just install `near` globally in image / pre-build anyway?

Reply:
I don't see what the problem is with this approach. Does this temporary Gitpod image really need a global dependency? The main thing is the command line acts the same as when they follow the more long-term instructions which tells them to install `near-shell`. Is this seen as blocking (a request to change) or a conversation about it?

Reply:
> The main thing is the command line acts the same as when they follow the more long-term instructions which tells them to install near-shell

So for this you want to have `yarn global add near-shell` in pre-build. Alias to locally installed behaves differently from globally installed. 

BTW, from my experience it doesn't work at all (raising question on how it was tested):
```
gitpod /workspace/near-chess $ near
bash: near: command not found
````

which makes sense, as `alias` is executed in context of building Docker image (and is active only in context of command passed with `bash -c`), not in context of GitPod's shell session.

Reply:
This is what I did in an example I worked on that worked well:
https://github.com/near-examples/token-contract-as/blob/master/.gitpod.yml#L26
I believe I tried installing globally but ran into issues, but I'll try again tomorrow.

Reply:
so yeah, that example does it in completely different place (not when building Docker image).

Reply:
The reason not to install globally is that we want to tie near-shell to the local version.

Reply:
` && alias near='yarn near'` in Dockerfile does nothing though @willemneal